### PR TITLE
ci: use find replace hardcode when modify func Version

### PR DESCRIPTION
**Pull request by j2gg0s**

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -53,26 +53,23 @@ if [[ "$(uname)" == "Darwin" ]]; then
...
-    sed -i "" "s/\(return \)\"[^\"]*\"/\1\"${TAG#v}\"/" ./dialect/mysqldialect/version.go
-    sed -i "" "s/\(return \)\"[^\"]*\"/\1\"${TAG#v}\"/" ./dialect/mssqldialect/version.go
-    sed -i "" "s/\(return \)\"[^\"]*\"/\1\"${TAG#v}\"/" ./dialect/pgdialect/version.go
-    sed -i "" "s/\(return \)\"[^\"]*\"/\1\"${TAG#v}\"/" ./dialect/sqlitedialect/version.go
-    sed -i "" "s/\(return \)\"[^\"]*\"/\1\"${TAG#v}\"/" ./dialect/oracledialect/version.go
+    for file in $(find . -type f -name 'version.go')
+    do
+        sed -i "" "/func Version() string/{n;s/\(return \)\"[^\"]*\"/\1\"${TAG#v}\"/;}" ${file}
+    done
     sed -i "" "s/\(\"version\": \)\"[^\"]*\"/\1\"${TAG#v}\"/" ./package.json
```
Comment (vmihailenco) [inline scripts/release.sh:60]: I believe we can use this version `sed -i` everywhere, i.e. it seems to be supported on Ubuntu/Linux.

Comment (j2gg0s) [reply scripts/release.sh:60]: @vmihailenco 

`sed -i ""` is feature of BSD, we must use `sed -i` in GNU

Comment (vmihailenco) [reply scripts/release.sh:60]: Alright, let's keep it as is then :+1:

Comment (vmihailenco) [reply scripts/release.sh:60]: Alright, let's keep it as is then :+1:

**Pull request merged**